firmware/console/binary/ts_can_channel.cpp

```diff
@@ -10,7 +10,7 @@ class CanTsChannel : public TsChannelBase {
 
 	// TsChannelBase implementation
 	void write(const uint8_t* buffer, size_t size) override;
-	size_t readTimeout(uint8_t* buffer, size_t size, int timeout) override;
+	size_t readTimeout(uint8_t* buffer, size_t size, sysinterval_t timeout) override;
```

Comment:
something is not right here

we change method signature only for ts_can_channel while not changing it for all implementations of ts channel

if we do a global rearch&replace we break unit tests which do not know about this ChibiOS type

since that's not important let's just undo this part of the change?

Reply:
Ups.
Anyway we pass this timeout to ChisiOS internals where `sysinterval_t` is expected. This type is unsigned 32bit.